suite-native/app/eas.json

```diff
@@ -32,10 +32,6 @@
     },
     "submit": {
         "develop": {
-            "android": {
```

Comment:
Why?

Reply:
It's not used since we deploy to firebase by a shell script.